nri/index.html

```diff
@@ -1 +1,298 @@
+        <ul>
+          <li>コンサルティング</li>
+          <li>金融ITソリューション</li>
+          <li>産業ITソリューション</li>
+          <li>IT基盤サービス</li>
+        </ul>
+      </div>
+    </section>
+
+    <div class="follow-us">
+      <ul>
+        <li class="letter">Follow Us</li>
+        <li><i class="fab fa-youtube fa-2x"></i></li>
+        <li><i class="fab fa-facebook-square fa-2x"></i></li>
+        <li><i class="fab fa-twitter fa-2x"></i></li>
+        <li><i class="fab fa-linkedin fa-2x"></i></li>
+      </ul>
+    </div>
+
+    <section class="bottom-index">
+      <div class="knowledge-insight">
+        <ul>
+          <li><p>ナレッジインサイト</p></li>
+          <li>刊行物</li>
+          <li>書籍</li>
+          <li>コラム</li>
+          <li>セミナー資料</li>
+          <li>レポート</li>
+          <li>用語解説</li>
+        </ul>
+      </div>
+
+      <div class="solution-service">
+        <ul>
+          <li><p>ソリューションサービス</p></li>
+          <li>マネジメントコンサルティング</li>
+          <li>システムコンサルティング</li>
+          <li>金融ITソリューション</li>
+          <li>産業ITソリューション</li>
+          <li>IT基盤サービス</li>
+          <li>ソリューション・製品・サービス</li>
+          <li>お客様事例</li>
+          <li>未来創発センター</li>
+        </ul>
+      </div>
+
+      <div class="sustainability">
+        <ul>
+          <li><p>サステナビリティ</p></li>
+          <li>サステナビリティマネジメント</li>
+          <li>取り組み</li>
+          <li>ライブラリ</li>
+        </ul>
+      </div>
+
+      <div class="news">
+        <ul>
+          <li><p>ニュース</p></li>
+          <li>ニュースリリース</li>
+          <li>お知らせ</li>
+          <li>イベント</li>
+        </ul>
+      </div>
+
+      <div class="about-company">
+        <ul>
+          <li><p>企業情報</p></li>
+          <li>社長メッセージ</li>
+          <li>経営方針</li>
+          <li>NRIの強み</li>
+          <li>企業案内</li>
+          <li>国内・海外拠点</li>
+        </ul>
+      </div>
+
+      <div class="shareholders-information">
+        <ul>
+          <li><p>株主・投資家情報（IR）</p></li>
+          <li>IRニュース</li>
+          <li>IRイベント</li>
+          <li>経営方針</li>
+          <li>財務・業績</li>
+          <li>IRライブラリ</li>
+          <li>株主・角付情報</li>
+          <li>個人投資家の皆様え</li>
+        </ul>
+      </div>
+
+      <div class="recruit-information">
+        <ul>
+          <li><p>採用情報</p></li>
+          <li><p>メディア関係者の皆様へ</p></li>
+          <li><p>ビジネスパートナーの皆様へ</p></li>
+        </ul>
+      </div>
+
+      <div class="nri-journal">
+        <ul>
+          <li><p>NRIJOURNAL</p></li>
+          <li><p>NRIPEOPLE</p></li>
+          <li><p>NRIvoice</p></li>
+        </ul>
+      </div>
+    </section>
+
+    <footer>
+      <div class="footer-content">
+        <ul>
+          <li>免責条項</li>
+          <li>サイト利用規定</li>
+          <li>個人情報保護方針</li>
+          <li>個人情報の取扱いについて</li>
+          <li>情報セキュリティ対策についての宣言文</li>
+          <li>お問い合わせ</li>
+          <li>アクセスマップ</li>
+        </ul>
+        <p>
+          <small
+            >Copyright(c) Nomura Research Institute, Ltd. All rights
+            reserved.</small
+          >
```

Comment:
smallタグの改行がおかしいので直しましょう

Reply:
改行が治っていません。修正するなら、こんな感じになります。

* smallタグでかこまれているよ、というのがわかりやすいパターン

```suggestion
          <small>
              Copyright(c) Nomura Research Institute, Ltd. All rightsreserved.
          </small>
```

* 短いから1行でやるパターン

```suggestion
          <small>Copyright(c) Nomura Research Institute, Ltd. All rightsreserved.</small>
```